src/main/java/io/github/cottonmc/cotton/mixin/ItemMixin.java

```diff
@@ -0,0 +1,21 @@
+package io.github.cottonmc.cotton.mixin;
+
+import io.github.cottonmc.cotton.interfaces.IChangeItemGroup;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemGroup;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+
+@Mixin(Item.class)
+public abstract class ItemMixin implements IChangeItemGroup {
```

Comment:
Sure they can. But we want to make sure that they can't forget to do so and end up using their own tab. Since that would influence not only their own mod, but all other mods that use the item too.

Reply:
And you know what they say about end users: "Never trust them to do things right."